tfe/resource_tfe_organization_test.go

```diff
@@ -29,6 +29,8 @@ func TestAccTFEOrganization_basic(t *testing.T) {
 						"tfe_organization.foobar", "email", "admin@company.com"),
 					resource.TestCheckResourceAttr(
 						"tfe_organization.foobar", "collaborator_auth_policy", "password"),
+					resource.TestCheckResourceAttr(
+						"tfe_organization.foobar", "owners_team_saml_role_id", "owners"),
```

Comment:
I noticed you didn't set this field in the config, so where does this value come from?

Reply:
I've removed it in bff01ae